src/main/java/frame/panels/PNLPanel.java

```diff
@@ -1,10 +1,40 @@
 package frame.panels;
 
 import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
 import java.awt.*;
 
 public class PNLPanel extends JPanel{
-    public PNLPanel(){
+    private final String[] currencyTypes;
+    public PNLPanel(String[] currencyTypes){
+        this.currencyTypes = currencyTypes;
+
+        createPNLTab();
```

Comment:
How is this going to be filled? Maybe we should include a reload button? Whatever we choose, I think we should include a function, which runs in the constructor, to fill it from the database.